packages/server/src/http.ts

```diff
@@ -0,0 +1,49 @@
+import { Deferred, Operation, Task, once } from 'effection';
+import { Express } from 'express';
+import type { AddressInfo } from 'net';
+
+export type { AddressInfo } from 'net';
+
+export interface Server {
+  listening(): Operation<AddressInfo>;
+}
+
+export interface Runnable<T> {
```

Comment:
should we put this in the `./interfaces` file it will be used in other places.

Reply:
Makes sense 👍🏻